### DISCUSSION TITLE
Alternative key bindings

### DISCUSSION THREAD

**Discussion by Lenbok** [ankitpokhrel/jira-cli#649] [Ideas]
As an emacs user, the navigation bindings in the TUI are unfamilar. Can I suggest the following bindings as a configurable alternative:

Ctrl-b,Ctrl-f,Ctrl-n,Ctrl-p (and arrow keys) - navigation
Alt-< - top of list
Alt-> - bottom of list
Ctrl-v (and page down key) - page down
Alt-v (and page up key) - page up
v / ENTER - view details (I think ENTER should be bound to something that stays in the TUI)
m - move
g / F5 - refresh
b - open in browser
u / Alt-w - copy issue URL to clipboard
k - copy issue key to clipboard
q - quit

**Comment by danpritts** [comment]
Better, allow the user to specify their key bindings in configuration.
